cypress/locators/menu/locators.js

```diff
@@ -3,3 +3,6 @@ export const SUBMENU = '[data-component="submenu-wrapper"]';
 export const SCROLL_BLOCK = '[data-component="submenu-scrollable-block"]';
 export const MENU_DIVIDER = '[data-component="menu-divider"]';
 export const SEGMENT_TITLE = '[data-component="menu-segment-title"]';
+export const MENU_ITEM = '[data-component="menu"]';
+export const MENU_ROOT = '[id="root"]';
+export const SUBMENU_ITEM = '[data-component="link"]';
```

Comment:
we don't need to define it here, because we can import the `link()` from `cypress/locators/link/index.js`